crcsim/experiment/combine_tests.py

```diff
@@ -0,0 +1,121 @@
+import json
+from pathlib import Path
+from typing import Dict
+
+import fire
+
+
+def combine_tests_in_params(
+    params: Dict, test1: str, test2: str, how: str = "serial"
+) -> Dict:
+    """
+    Combines two tests in the parameters dictionary and adds the combined test.
+    Returns: The modified parameters dictionary
+    """
+    if test1 not in params["tests"]:
+        raise ValueError(f"Test '{test1}' not found in parameters")
+    if test2 not in params["tests"]:
+        raise ValueError(f"Test '{test2}' not found in parameters")
+    if how not in ["serial", "parallel"]:
+        raise ValueError("how must be 'serial' or 'parallel'")
+
+    t1 = params["tests"][test1]
+    t2 = params["tests"][test2]
+    combined = {}
+
+    # Combine sensitivity and specificity for all relevant fields
+    for key in [
+        "sensitivity_polyp1",
+        "sensitivity_polyp2",
+        "sensitivity_polyp3",
+        "sensitivity_cancer",
+    ]:
+        if how == "serial":
+            combined[key] = t1[key] * t2[key]
+        elif how == "parallel":
+            combined[key] = 1 - (1 - t1[key]) * (1 - t2[key])
+
+    if how == "serial":
+        combined["specificity"] = t1["specificity"] * t2["specificity"]
+    else:
+        combined["specificity"] = 1 - (1 - t1["specificity"]) * (1 - t2["specificity"])
+
+    # Combine cost and other fields
+    combined["cost"] = t1.get("cost", 0) + t2.get("cost", 0)
+    combined["proportion"] = 1.0  # Set to 1.0 to ensure it's selected
+    combined["routine_start"] = min(
+        t1.get("routine_start", 45), t2.get("routine_start", 45)
+    )
+    combined["routine_end"] = max(t1.get("routine_end", 75), t2.get("routine_end", 75))
+    combined["routine_freq"] = min(t1.get("routine_freq", 1), t2.get("routine_freq", 1))
```

Comment:
This is interesting, I hadn't thought about the trickiness of combining these params. 

It would give us more flexibility to allow specifying some params manually as CLI arguments to `main`. But I think this is probably okay for now. We can always change them in specific scenarios if needed.

Reply:
But as above, let's just take the max/min without defaults.